### PR TITLE
NXT-480 action banners for category landing page

### PR DESCRIPTION
**Pull request by johndavey72**
https://nicedigital.atlassian.net/browse/NXT-480

### CODE REVIEW

File: web/src/pages/[...slug].page.tsx [hunk 1]
```diff
@@ -171,6 +181,7 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
 			siblingPages.push(...["page1", "page2"]);
 		}
 
+		// console.log(JSON.stringify(storyResult, null, 2));
```
Comment (johnholland-nice) [inline web/src/pages/[...slug].page.tsx:184]: Think this needs to be removed?

File: web/src/components/Storyblok/CategoryLandingPage/CategoryLandingPage.test.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+describe("CategoryLandingPage", () => {
+	it("includes the metadata through the StoryblokComponent", () => {
+		render(<CategoryLandingPage {...mockPropsWithHero} />);
+		expect(
+			screen.getByText(mockHeroData.metadata[0].component)
+		).toBeInTheDocument();
+	});
+
+	xit("renders the Hero through the StoryblokComponent, if it's passed as block", () => {
```
Comment (johnholland-nice) [inline web/src/components/Storyblok/CategoryLandingPage/CategoryLandingPage.test.tsx:42]: Does xit run a single test? Guessing this should just be it?

File: web/src/components/Storyblok/CategoryLandingPage/CategoryLandingPage.test.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+		render(<CategoryLandingPage {...mockPropsWithHero} />);
+		expect(
+			screen.getByText(mockHeroData.header[0].component)
+		).toBeInTheDocument();
+	});
+
+	xit("renders the Page Header through the StoryblokComponent, if it's passed as block", () => {
```
Comment (johnholland-nice) [inline web/src/components/Storyblok/CategoryLandingPage/CategoryLandingPage.test.tsx:49]: Same as above.

**Review by johnholland-nice — COMMENTED** [review]
Just a couple of bits that I think you left in by accident. :)

**Review by johnholland-nice — APPROVED** [review]

**Review by johnholland-nice — APPROVED** [review]

**Review by tresaknice — APPROVED** [review]

**Pull request merged**